client/layout/index.jsx

```diff
@@ -167,6 +168,7 @@ Layout = React.createClass( {
 					isActive={ translator.isActivated() } />
 				{ this.renderPreview() }
 				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="components/happychat" /> }
+				<AppBanner />
```

Comment:
Let's 🏃  with this. Let's make sure we do an very early return if the section doesn't match though.